src/server/rpc/procedures/NPlayer/NPlayer.js

```diff
@@ -0,0 +1,151 @@
+// This is the NPlayer game RPC. It ensures round-robin turn taking with N players.
+
+'use strict';
+
+var R = require('ramda'),
+    debug = require('debug'),
+    log = debug('NetsBlox:RPCManager:NPlayer:log'),
+    trace = debug('NetsBlox:RPCManager:NPlayer:trace'),
+    Constants = require('../../../../common/Constants'),
+    info = debug('NetsBlox:RPCManager:NPlayer:info');
+
+/**
+ * NPlayerRPC - This constructor is called on the first request to an RPC
+ * from a given room
+ *
+ * @constructor
+ * @return {undefined}
+ */
+var NPlayerRPC = function() {
```

Comment:
Do you mind changing the name of this to just `NPlayer`? It should be clear that it is an RPC from the location and comments, etc.